R/simulate_cord.R

```diff
@@ -1,19 +1,35 @@
 #'Simulates new data from a given cord object
 #'
-#' @param obj is a cord object, e.g. from output of \code{cord}
-#' @param nsim number of simulations, defaults to 1
-#' @param seed random number seed
+#' @param object is a cord object, e.g. from output of \code{cord}
+#' @param nsim Number of simulations, defaults to 1.
+#' @param seed Random number seed, defaults to a random seed number.
+#' @param newdata A data frame in which to look for X covariates with which to simulate.
+#' Defaults to the X covariates in the fitted model.
+#' @param reshape2matrix logical. Returns a matrix of simulated data if TRUE or nsim == 1.
+#' If TRUE and nsim > 1, the simulated data will be appended and returned as a matrix.
+#' Returns a 3-d array of simulated data if FALSE and nsim > 1. Each simulation is returned
+#' along the third dimension. Defaults to TRUE.
+#' @param coeffs Coefficient matrix for a `manyglm` object that characterises the size of effects
```

Comment:
see lme4 newparams

Reply:
`coeffs` arg is no longer required as `predict.manyglm` allows for changing `coeffs` now.